pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -222,6 +222,17 @@ type MetricHTTP struct {
 	// If false (the default), metrics for the HTTP server are collected with increased cardinality.
 	// +optional
 	IncreasedCardinality *bool `json:"increasedCardinality,omitempty"`
+	// +optional
+	PathNormalization *PathNormalization `json:"pathNormalization,omitempty"`
+}
+
+type PathNormalization struct {
+	// If enabled, the path normalization will be applied to the HTTP server metrics.
+	Enabled *bool `json:"enabled"`
```

Comment:
No need for `*bool` since the default is false, just `bool`

Please don't forget `,omitempty` here and below

Reply:
I think `*bool` would help us in case we want to change the default value, this way we always fallback to false.

Anyway, I removed it and added a test for the default case when pathNormalization is not in the spec.